Dockerfile

```diff
@@ -5,6 +5,14 @@ FROM ruby:2.4.2
 RUN mkdir /app
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y wget
```

Comment:
wget is already installed in the ruby:2.4.2 image.

Reply:
removed